code/modules/clothing/neck/_neck.dm

```diff
@@ -120,6 +128,8 @@
 
 /obj/item/clothing/neck/tie/add_context(atom/source, list/context, obj/item/held_item, mob/user)
 	. = ..()
+	if(!isnull(held_item))
```

Comment:
You only add the alt-click context if `held_item` exists, but I believe your `alt_click_secondary(...)` doesn't actually care about whether you have an item in your active hand or not?

Reply:
IIRC I think I only implemented it that way because other things that similarly didn't care also checked, but yeah, this isn't necessary
You also made me notice I somehow got the check backwards but I guess that doesn't matter when I'm killing it anyway, lol

Reply:
yeah I noticed cause I was killing this elsewhere lmao